packages/react-form/src/hooks/form.ts

```diff
@@ -119,14 +124,17 @@ export function useForm<T extends FieldBag, D extends DynamicListBag>({
 
   const fieldsRef = useLazyRef(() => fieldsWithLists);
   fieldsRef.current = fieldsWithLists;
+  const dynamicListsRef = useLazyRef(() => dynamicLists);
+  dynamicListsRef.current = dynamicLists;
```

Comment:
This should set `current` on line 127 no?

Reply:
Yeah sorry this: `const dynamicListsRef = useLazyRef(() => dynamicLists);`
and this `const dynamicListsRef = useRef(); dynamicListsRef.current = dynamicLists;`  are equivalent from what I see. You should just be able to do `const dynamicListsRef = useLazyRef(() => dynamicLists);` without assign it to `.current`.

Reply:
Ahh I see, yes it should typically set the current value when passed in. Will change it for both! thanks :)